public/index.html

```diff
@@ -22,14 +22,19 @@
   <meta property="twitter:image" content="https://twin-cities-mutual-aid.org/images/aid2.png">
 </head>
 
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-9313001-2"></script>
```

Comment:
@crhistianramirez Did you add this Google Tag Manager or did someone else? I think Google Tag Manager is sometimes used to add and manage adding multiple sets of analytic like scripts. If someone else added this and they expected to manage things through Google Tag Manager, I'd want to make sure we communicated this change.

Reply:
@crhistianramirez It looks like you've traded out the entirety of Google Tag Manager for Google Analytics' auto-tracking of changed URLs. I think some of the folks over at Data Entry are using the tag manager as they requested I set it up for them so they could manage the tags. I'm not sure that's a great idea. While I like the auto-tracking of auto-changed URLs on searches, this is almost surely going to break something that someone's using.

Reply:
What we had previously is gtag.js which is essentially a wrapper to analytics.js, not google tag manager. For comparison [here](https://developers.google.com/tag-manager/quickstart) is what a script for google tag manager would look like. 

It is pretty confusing (there's actually 3 analytics implementations!), [this article](https://www.watermelonwebworks.com/is-gtag-js-the-same-as-google-tag-manager/) kind of helped clear things up for me. The reason I changed it from using gtag.js to the more legacy analytics.js because that is what is compatible with autotrack and is recommended by the [autotrack community](https://github.com/googleanalytics/autotrack/issues/202#issuecomment-333744194)

Autotrack seemed like a neat library but I'm totally fine with just not including it and doing it the more manual way.